stream-chat-android-client/src/main/java/io/getstream/chat/android/client/utils/internal/toggle/dialog/ToggleAdapter.kt

```diff
@@ -28,7 +28,9 @@ internal class ToggleAdapter(context: Context) :
         val itemView =
             convertView ?: LayoutInflater.from(parent.context).inflate(R.layout.stream_toggle_list_item, parent, false)
         val label: TextView = itemView.findViewById(R.id.label)
-        val switch: SwitchCompat = itemView.findViewById(R.id.switcher)
+        val switch: SwitchCompat = itemView.findViewById<SwitchCompat>(R.id.switcher).also {
+            it.setOnCheckedChangeListener(null)
```

Comment:
when a view isn't null we already have a listener. We need to set it null to avoid firing it when set a value